extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -71,40 +91,50 @@ export class AmpAnalytics extends AMP.BaseElement {
      */
     this.remoteConfig = {};
 
-    return this.fetchRemoteConfig_().then(() => {
-      /**
-       * @private {!JSONObject} The analytics config associated with the tag
-       */
-      this.config_ = this.mergeConfigs_();
+    return this.consentPromise_.then(() => {
+      return this.fetchRemoteConfig_()
+          .then(this.onFetchRemoteConfigSuccess_.bind(this));
+    });
```

Comment:
This can be written without the nesting:

```js
return this.consentPromise_
  .then(this.fetchRemoteConfig_.bind(this)
  .then(this.onFetchRemoteConfigSuccess_.bind(this));